custom-recipes/ml-assisted-labeling-deephub-export/recipe.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import dataiku
+from dataiku.customrecipe import get_recipe_config, get_input_names_for_role, get_output_names_for_role
+
+
+def format_labeling_plugin_annotations(image_annotations):
+    """ Format labeling annotations to be compatible with dss object detection (deephub)"""
+
+    deephub_image_annotations = []
+    # if images were skipped during labeling plugin, their annotations is nan:
+    image_annotations = json.loads(image_annotations) if isinstance(image_annotations, str) else []
+    for annotation in image_annotations:
+        deephub_image_annotations.append(
+            {"bbox": [annotation["left"], annotation["top"], annotation["width"], annotation["height"]],
+             "category": annotation["label"]
+             })
+    image_annotations = json.dumps(deephub_image_annotations)
+    return image_annotations
+
+
+dataset_df = dataiku.Dataset(get_input_names_for_role("input_dataset")[0]).get_dataframe()
+
+target_column = get_recipe_config()["target_column"]
+if target_column not in dataset_df.columns:
+    raise Exception("Column {} not in dataset with columns [{}]".format(target_column, dataset_df.columns.tolist()))
```

Comment:
No need for the extra brackets
```suggestion
    raise Exception("Column {} not in dataset with columns {}".format(target_column, dataset_df.columns.tolist()))
```